pithos/pithos.py

```diff
@@ -843,7 +845,18 @@ def update_song_row(self, song = None):
         if song:
             self.songs_model[song.index][1] = self.song_text(song)
             self.songs_model[song.index][2] = self.song_icon(song) or ""
-        return self.playing
+        if self.playing:
+            return True
+
+    def create_ui_loop(self):
+        if self.ui_loop_timer_id is not None:
```

Comment:
I just used what the Exaile devs used. They used None. I haven't tried to modify that part of the code as it has NEVER failed.